images/runtime/dotnetcore/generateDockerfiles.sh

```diff
@@ -21,8 +21,8 @@ declare -r DOCKERFILE_TEMPLATE="$DIR/template.Dockerfile"
 declare -r RUNTIME_BASE_IMAGE_TAG_PLACEHOLDER="%RUNTIME_BASE_IMAGE_TAG%"
 declare -r BASE_IMAGE_REPO_PLACEHOLDER="%BASE_IMAGE_REPO%"
 declare -r DOTNET_VERSION_PLACEHOLDER="%DOTNET_VERSION%"
-declare -r NETCORE_BUSTER_VERSION_ARRAY=($NET_CORE_APP_30 $NET_CORE_APP_50 $NET_CORE_APP_60 $NET_CORE_APP_70)
-declare -r NETCORE_BULLSEYE_VERSION_ARRAY=($NET_CORE_APP_31)
+declare -r NETCORE_BUSTER_VERSION_ARRAY=($NET_CORE_APP_30 $NET_CORE_APP_31 $NET_CORE_APP_50 $NET_CORE_APP_60 $NET_CORE_APP_70)
```

Comment:
That being said, if it is a permanent switch to buster and we no longer are releasing any .NET runtimes with bullseye, we should delete the below line that's initializing an empty array for bullseye, and update the release flow to prevent it from trying to release any .NET runtimes for bullseye:

- [Block to release bullseye base images for .NET](https://github.com/microsoft/Oryx/blob/main/vsts/pipelines/baseImages/dotnetcore.yml#L17-L24)
- [Line forcing the next job to depend on this bullseye job to finish before executing](https://github.com/microsoft/Oryx/blob/main/vsts/pipelines/baseImages/dotnetcore.yml#L29)
- [Line that tags bullseye base images for .NET](https://github.com/microsoft/Oryx/blob/main/vsts/scripts/tagBaseImagesForRelease.sh#L98)

(The lines linked above should be removed in this case)

Reply:
@cormacpayne Yes, it's a one time need to get 3.1 Buster latest version. After that, we will never update 3.1 anymore (as it is EOL and that both Buster and Bullseye will already be available for the latest version).

Reply:
Both .NET 6 and 7 are currently based on Buster, but I would still keep the Bullseye logic around for when we transition them in the future.